diagnostic.md

```diff
@@ -40,31 +52,46 @@ like to show all movies favorited by a profile on
 `http://localhost:3000/profiles/1`
 
   ```md
-    # < Your Response Here >
+    The purpose of a serializer is to take the requested data from the database
+    and turn it into a json string.
```

Comment:
Yes but an important part is that it determines how the string is returned; it filters the JSON so we can control the what data is sent back from the server.